src/chainstate/stacks/boot/mod.rs

```diff
@@ -124,7 +131,340 @@ pub fn make_contract_id(addr: &StacksAddress, name: &str) -> QualifiedContractId
     )
 }
 
+/// Extract a PoX address from its tuple representation
+fn tuple_to_pox_addr(tuple_data: TupleData) -> (AddressHashMode, Hash160) {
```

Comment:
I deleted this in my native-segwit branch because it's only used in tests below (where it is duplicated).  I think its presence here is a merge artifact?

Reply:
Yep, a merge artifact, removed in 9d00a6cfd48fe557dc524a6935e7ab9c1af74d79